lib/src/replaced_element.dart

```diff
@@ -230,22 +231,24 @@ class EmptyContentElement extends ReplacedElement {
 class RubyElement extends ReplacedElement {
   dom.Element element;
 
-  RubyElement({required this.element, String name = "ruby"})
-      : super(name: name, alignment: PlaceholderAlignment.middle, style: Style(), elementId: element.id);
+  RubyElement({
+    required this.element,
+    required List<StyledElement> children,
+    String name = "ruby"
+  }) : super(name: name, alignment: PlaceholderAlignment.middle, style: Style(), elementId: element.id, children: children);
 
   @override
   Widget toWidget(RenderContext context) {
-    dom.Node? textNode;
+    String? textNode;
     List<Widget> widgets = <Widget>[];
-    //TODO calculate based off of parent font size.
```

Comment:
😬

Reply:
Should we really calculate based off parent font size though? I feel that calculating based on whatever styling is set on the `ruby` tag should be sufficient, and now we even allow for specific styling on `rt` itself.

Reply:
I agree.